export.js

```diff
@@ -135,6 +135,9 @@ Avatar = {
   // Get the url of the user's avatar
   getUrl: function (user) {
 
+    // Default to the currently logged in user, unless otherwise specified.
+    if (!user) user = Meteor.users.findOne(Meteor.userId())
```

Comment:
@timothyarmes Hmm, good point again! But, will this mean that renders (when a user isn't supplied to `Avatar.getUrl`) will happen a lot more often if anything in the `Meteor.user()` object changes, even if not related to the avatar?